Framework/DataHandling/inc/MantidDataHandling/LoadANSTOHelper.h

```diff
@@ -135,17 +135,15 @@ class EventAssignerFixedWavelength : public EventAssigner {
   double m_wavelength;
 
   // methods
-  void addEventImpl(size_t id, double tof) override;
+  void addEventImpl(size_t id, int64_t pulse, double tof) override;
 
 public:
   // construction
-  EventAssignerFixedWavelength(const std::vector<bool> &roi,
-                               const size_t stride, const double wavelength,
-                               const double period, const double phase,
-                               const double tofMinBoundary,
-                               const double tofMaxBoundary,
-                               const double timeMinBoundary,
-                               const double timeMaxBoundary,
+  EventAssignerFixedWavelength(const std::vector<bool> &roi, size_t stride,
+                               double wavelength, double period, double phase,
+                               int64_t startTime, double tofMinBoundary,
+                               double tofMaxBoundary, double timeMinBoundary,
+                               double timeMaxBoundary,
```

Comment:
All the const's on input parameters have been removed from the functions here. Would it be possible to only remove the const qualifier from inputs which need to be modified and leave it on the rest?

Reply:
Generally, I don't bother including a const for pass by value arguments. Is this a coding standard for Mantid, I can change it but it doesn't seem to be applied globally across the project? 

I have one request - could we hold off closing the review before I speak with the instrument scientist, who is returning from holidays on Monday, just to avoid an additional minor pull request cycle.

Reply:
It's not explicitly stated in the standards, but I think it's generally good practice to avoid accidental errors. It's not applied throughout the project, however, because there is the argument for keeping the coding style of the file/directory when writing new functions, so files without const often remain without const

We have a `State: In Progress` label for PRs which means "Don't review / merge yet"

Reply:
I've restored the const values as the file did include them initially. My personal preference is not to use const because it clutters the definition without any significant gain. Most library definitions and cppcheck do not identify the need for const for pass by value arguments. Anyway it's done. 

I've cleared the changes with the scientist so the PR doesn't need to be suspended. Thanks.